releasenotes/notes/fix-circuit-drawer-for-qc-params-e78c67310ae43ccf.yaml

```
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    Fixes the circuit drawer from displaying parameters when they are of type `QuantumCircuit` 
```

```suggestion
    Fixes the circuit drawer from displaying parameters when they are of type :class:`.QuantumCircuit` 
```

this makes it a Sphinx cross-reference, so the link will be clickable